Detections/MultipleDataSources/SucessfullSiginFromPhingLink.yaml

```diff
@@ -0,0 +1,143 @@
+id: 779731f7-8ba0-4198-8524-5701b7defddc
+name: M365D Alerts Correlation to non microsoft Network device network activity involved in successful signing Activity
```

Comment:
Microsoft should have a capital M at all time.

Reply:
signing Activity -> sign-in activity

Reply:
Addressed